build.gradle

```diff
@@ -104,8 +104,9 @@ processResources.dependsOn writeBuildProperties
 build.dependsOn shadowJar, groovydoc, cloneContribFiles
 
 // FIXME: use ShadowJar (java -jar build/libs/web-api-skeleton-all.jar server configuration.yaml)
-task run(type: JavaExec, dependsOn: build) {
+task run(type: JavaExec) {
```

Comment:
Did you remove the build dependency just for testing?

Reply:
Right. Run doesn't really depend on build, so removing it speeds things up significantly.

I didn't mean to commit it though; thanks for catching.